ethcore/src/client/client.rs

```diff
@@ -899,6 +900,7 @@ impl BlockChainClient for Client {
 		}
 		let options = TransactOptions { tracing: true, vm_tracing: false, check_nonce: false };
 		let mut tx = t.clone();
+		tx.gas_price = 0.into();
```

Comment:
It affects the value that can be transfered (lower by `gas*gas_price`) and also may affect executed code (`GASPRICE` opcode)